src/mudlet.cpp

```diff
@@ -1579,13 +1507,16 @@ void mudlet::addConsoleForNewHost(Host* pH)
     mpTabBar->setTabData(newTabID, tabName);
 
     //update the main window title when we spawn a new tab
-    setWindowTitle(pH->getName() + " - " + version);
+    setWindowTitle(tr("%1 - %2",
```

Comment:
The spaces and the hyphen are probably not appropriate for some Far Eastern locales...!

It is also possible that the ordering might not be the same everywhere, i.e. the version number might be first in some places. :man_shrugging:

Reply:
I'm not certain about that... lets leave it be for now until folks ask and we know it's needed for sure.

Reply:
Okay - I guess - for NOW.

:earth_africa:  **I18n**/**L10n** I say NOW because we currently force the user to call their profiles something that only uses a subset of ASCII characters (so that it is safe to use them as a directory name) - but long-term we ought to try and find a way around that so that a profile can be named in an end-user locale appropriate manner.